tox.ini

```diff
@@ -0,0 +1,22 @@
+[tox]
+env_list = py312,black,flake8
+
+[testenv]
+use_develop = True
+deps =
+    -r requirements.txt
+    -r requirements-test.txt
+commands = python3 -m pytest --cov-report=html --cov-report=term --cov=pipeline_migration {posargs:tests/}
+
+[testenv:black]
+deps = black
+skip_install = True
+commands = python3 -m black --line-length 100 --check --diff {posargs:src/ tests/}
+
+[testenv:flake8]
+deps = flake8
+skip_install = True
+commands = python3 -m flake8 {posargs:src/ tests/}
+
+[flake8]
+max-line-length = 100
```

Comment:
Is it possible to set this in pyproject.toml instead? To make sure that when I run `flake8` directly instead of using tox, it uses the correct settings (or when my editor runs flake8 automatically)

Reply:
Ah nevermind, `flake8` supports reading the config from tox.ini. I though it would only work when running through tox, but I don't think that's the case